en/docs/observability/root-cause-analysis.md

```diff
@@ -0,0 +1,99 @@
+# Root Cause Analysis
+
+Performing root cause analysis is crucial in identifying and rectifying the underlying problem for any anomalies detected in a system. Choreo Observability provides helpful insights out-of-the-box that assists in carrying out a root cause analysis for any service deployed on the Choreo platform.
+
+!!! info
+    For explanation, we are using the sample data available in the product.
+    
+## Get started with the sample service
+
+1. Go to **Services** from the left navigation.
+2. If this is your first service, then enter a name for the service and click **Create**. Skip step 3 and go to step 4.
+       ![Create a new service](/assets/img/observability/first-service-creation.png)
+3. Else, click **+ Create** to create a new service.
+       ![Create a new service](/assets/img/observability/service-creation.png)
+    1. Enter a name for the service and click **Create**.
```

Comment:
Is this number one needed as there is only one sub item? If we really need it shall we use roman numbers for sub numbers?

Reply:
This will appear as "a". If not, it will not appear as a sub step, isn't it? do you think we should just add a bullet point?

Reply:
I think it will appear as roman number one. But for me having just one number gave me the feeling other numbers are missing in the doc. But if this is the convention even for one item that will be ok.